ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml

```diff
@@ -41,1759 +41,1639 @@ topologies:
     priority: 50
     job:
       class: RunPytest
       args:
-        build_url: '{testing-fedora/build_url}'
-        update_packages: True
+        build_url: '{fedora-previous-ipa-4-9/build_url}'
         test_suite: test_integration/test_automember.py
-        template: *testing-master-latest
+        template: *ci-ipa-4-9-previous
         timeout: 3600
         topology: *master_1repl
 
-  testing-fedora/test_crlgen_manage:
-    requires: [testing-fedora/build]
+  fedora-previous-ipa-4-9/test_crlgen_manage:
+    requires: [fedora-previous-ipa-4-9/build]
     priority: 50
     job:
       class: RunPytest
       args:
-        build_url: '{testing-fedora/build_url}'
-        update_packages: True
+        build_url: '{fedora-previous-ipa-4-9/build_url}'
         test_suite: test_integration/test_crlgen_manage.py
-        template: *testing-master-latest
+        template: *ci-ipa-4-9-previous
         timeout: 3600
         topology: *master_1repl
 
-  testing-fedora/test_integration_TestIPANotConfigured:
-    requires: [testing-fedora/build]
+  fedora-previous-ipa-4-9/test_integration_TestIPANotConfigured:
+    requires: [fedora-previous-ipa-4-9/build]
     priority: 50
     job:
       class: RunPytest
       args:
-        build_url: '{testing-fedora/build_url}'
-        update_packages: True
+        build_url: '{fedora-previous-ipa-4-9/build_url}'
         test_suite: test_integration/test_cli_ipa_not_configured.py::TestIPANotConfigured
-        template: *testing-master-latest
+        template: *ci-ipa-4-9-previous
         timeout: 10800
         topology: *master_1repl
 
-  testing-fedora/test_sssd:
-    requires: [testing-fedora/build]
+  fedora-previous-ipa-4-9/test_sssd:
+    requires: [fedora-previous-ipa-4-9/build]
     priority: 50
     job:
       class: RunADTests
       args:
-        build_url: '{testing-fedora/build_url}'
-        update_packages: True
+        build_url: '{fedora-previous-ipa-4-9/build_url}'
         test_suite: test_integration/test_sssd.py
-        template: *testing-master-latest
+        template: *ci-ipa-4-9-previous
         timeout: 7200
         topology: *adroot_adchild_adtree_master_1client
 
-  testing-fedora/test_ca_custom_sdn:
-    requires: [testing-fedora/build]
+  fedora-previous-ipa-4-9/test_ca_custom_sdn:
+    requires: [fedora-previous-ipa-4-9/build]
     priority: 50
     job:
       class: RunPytest
       args:
-        build_url: '{testing-fedora/build_url}'
-        update_packages: True
+        build_url: '{fedora-previous-ipa-4-9/build_url}'
         test_suite: test_integration/test_ca_custom_sdn.py
-        template: *testing-master-latest
+        template: *ci-ipa-4-9-previous
         timeout: 7200
         topology: *master_1repl
 
-  testing-fedora/test_membermanager:
-    requires: [testing-fedora/build]
-    priority: 50
+  fedora-previous-ipa-4-9/test_membermanager:
+    requires: [fedora-previous-ipa-4-9/build]
+    priority: 100
```

Comment:
usually tests are run with priority: 50, was it a mistake?

Reply:
Same as https://github.com/freeipa/freeipa/pull/5280#discussion_r559686065: 
> I've copied the content from master branch's nightly_latest.yaml, no idea why there were set with 100 there.
PR-CI will execute them before the rest of the tests, that would be the only "side effect".